ONSdigital/rm-common-service
# Improve logging

# Motivation and Context
Initially this change was to improve the logging in
RestExceptionHandler as we were logging without adding context such as
this example ""event":"handleInvalidRequestException {}". With
https://github.com/ONSdigital/sdc/pull/4 it made sense to also look at
using those logging standards

# What has changed
* Using godaddy logger
* Improved logging details/removing unnecessary logging

# How to test?
* Run `mvn install` and check for logs from tests

```
[WARN] No valid transition; event=REQUEST_ACCEPTED; from=SUBMITTED
[INFO] Transitioning state; event=REQUEST_DISTRIBUTED; from=SUBMITTED; to=PENDING
```
